src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -87,43 +81,84 @@ function BaseVideoPlayer({
         opacity: controlsOpacity.get(),
     }));
 
-    const videoPlayerRef = useRef<VideoWithOnFullScreenUpdate | null>(null);
+    /* eslint-disable no-param-reassign */
```

Comment:
Could you please always add explanation of why the rule has to be disabled and there is no reasonable way to do it otherwise?

Reply:
You can add it in a follow up.